File: Golang/gin-gorm-songs-api/models/entities/artist.go
package entities

import (
	"gorm.io/gorm"
	"time"
)

type Artist struct {
	ID        string         `gorm:"primary_key;type:uuid"`
	Name      string         `gorm:"type:varchar(100);not null"`
	Bio       string         `gorm:"type:text"`
	CreatedAt time.Time      `gorm:"default:CURRENT_TIMESTAMP"`
	UpdatedAt time.Time      `gorm:"default:CURRENT_TIMESTAMP"`
	DeletedAt gorm.DeletedAt `gorm:"index"`
}

File: Python/django-songs-api/api/models/artist.py
from django.db import models


class Artist(models.Model):
    id = models.AutoField(primary_key=True)
    name = models.CharField(max_length=255)
    bio = models.TextField(blank=True, null=True)
    created_at = models.DateTimeField(auto_now_add=True, null=True, editable=False)
    updated_at = models.DateTimeField(auto_now=True, null=True, editable=False)

    def __str__(self):
        return self.name
